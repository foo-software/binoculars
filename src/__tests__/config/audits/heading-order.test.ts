import auditHeadingOrder from '../../../config/audits/heading-order';

describe('heading-order audit', () => {
  it('should match snapshot', () => {
    expect(auditHeadingOrder('en')).toMatchSnapshot();
  });
});
