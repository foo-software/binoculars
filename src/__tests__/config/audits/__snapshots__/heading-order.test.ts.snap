// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`heading-order audit should match snapshot 1`] = `
Object {
  "description": "Properly ordered headings that do not skip levels convey the semantic structure of the page, making it easier to navigate and understand when using assistive technologies. [Learn more](https://web.dev/heading-order/).",
  "title": "Heading elements appear in a sequentially-descending order",
}
`;
